packages/core/core/src/RequestTracker.js

```diff
@@ -557,7 +558,7 @@ export class RequestGraph extends ContentGraph<
       // If we see a 'create' event for the project root itself,
       // this means the project root was moved and we need to
       // re-run all requests.
-      if (type === 'create' && filePath === '') {
+      if (opts?.isInitialBuild && type === 'create' && filePath === '') {
```

Comment:
I think @devongovett was suggesting lifting this to where we load an existing RequestGraph to eliminate the flag. Would need to iterate events there too in order to find this one though — I'm fine either way.